advanced_functionality/pytorch_yolov5_training_and_hpo/transfer_learning_and_hpo_yolov5.ipynb

```diff
@@ -0,0 +1,1379 @@
+{
```

Comment:
<p>Did you submit a request for dataset review by legal?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3601/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
No, i haven't. How do I go about it? Any guidance on a Wiki?

Reply:
<p>Just for the record, the request to legal was submitted, and legal approved it. TT URL: <a href="https://t.corp.amazon.com/D57175372/overview" target="_blank">https://t.corp.amazon.com/D57175372/overview</a></p><p> </p>